infrastructure/src/test/java/mops/controllers/StudentControllerTest.java

```diff
@@ -89,4 +88,11 @@ public void correctRedirectForFeedbackPost() throws Exception {
         .andExpect(status().is3xxRedirection())
         .andExpect(view().name("redirect:/feedback/student/"));
   }
+
+  @Test
+  @DisplayName("Student sollte nicht auf die Orga Uebersicht Seite kommen")
+  @WithMockKeycloackAuth(roles = userrole, idToken = @WithIDToken(email = usermail))
+  public void forbiddenAccessStudent() throws Exception {
+    mvc.perform(get("/feedback/dozenten")).andExpect(status().is4xxClientError());
+  }
```

Comment:
You probably have to change this again after merging with the error Page Branch

Reply:
Yes, all the tests will updated as we move along.